components/ProductCollection/ProductCollection.tsx

```diff
@@ -75,14 +75,17 @@ export function ProductCollection({
   const products = mapEdgesToItems(data?.products);
   if (products.length === 0) {
     return (
-      <Text size="xl" color="secondary">
+      <Text size="xl" color="secondary" data-testid="noResultsText">
         {t.formatMessage(messages.noProducts)}
       </Text>
     );
   }
   return (
     <div>
-      <ul className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-12">
+      <ul
+        className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-12"
+        data-testid="productCollection"
+      >
         {products.map((product) => (
           <ProductCard key={product.id} product={product} />
```

Comment:
missed space after < , or remove space before />

Reply:
I can but in this line I didn't change anything :)

Reply:
I can't change it pnpm lint automatically restores those spaces :D